src/jit/flowgraph.cpp

```diff
@@ -23074,6 +23087,8 @@ GenTreePtr Compiler::fgInlinePrependStatements(InlineInfo* inlineInfo)
                 if (argInfo.argHasSideEff)
                 {
                     noway_assert(argInfo.argIsUsed == false);
+                    newStmt       = nullptr;
+                    bool noAppend = false;
```

Comment:
Suggestion: I think it's always better to avoid a double negative, so I would name this `append` and initialize it to true.

Reply:
Fixed.